pagopa/io-app
# chore(Cross): [IOAPPX-295] Remove nested navigator from DS section

## Short description
This PR removes the nested navigator from the DS section thanks to the upgrade to `react-navigation` v.6.x. This also fixes the warning that appeared every time you entered the section.

## List of changes proposed in this pull request
- Remove nested navigator from `navigator`
- **[extra]** Decrease top margin in the `DebugInfoOverlay` to avoid overlapping information when debug mode is enabled
- **[extra]** Refactor `Overlay` legacy component:
  - converted from class component to functional one
  - fixed missing margin

## How to test
1. Launch the app in the local environment
2. Go to the **Design System** section
3. The warning message at the bottom of the page should no longer appear

#### Related PR
* https://github.com/pagopa/io-app/pull/5415
